packages/manager/apps/dedicated/client/app/license/license.controller.js

```diff
@@ -171,7 +176,20 @@ export default /* @ngInject */ (
     $state.go('app.license.order');
   };
 
+  const checkCpanel = () => {
+    const eolDate = new Date('2023-07-31');
```

Comment:
**nitpick:** Use `feature-flipping` to manage dates / enable / disable banner and so on.